go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gabriel-vasile/mimetype v1.2.0
 	github.com/google/go-containerregistry v0.5.0
 	github.com/onsi/gomega v1.11.0
+	github.com/paketo-buildpacks/occam v0.1.2
```

Comment:
So, I didn't think about it at the time you asked me, but this creates a dependency loop. `packit` depends on `occam` which depends on `packit`, and on and on. It means that we will just have releases that create dependabot PRs and thus unreleased commits requiring new releases, ad infinitum.

Reply:
Hmmm, ok I will take a look at this then. I might just steal the matcher and put it in `packit` as well

Reply:
I added the ContainLines matcher to the `packit` matchers. If this does not work for you I can look into doing something else.